### ISSUE TITLE
Django compatibility issues

### ISSUE THREAD

**Issue by pisapapiros** [django-oscar/django-oscar-paypal#157]
Current version throws the following error 

```
lib\site-packages\paypal\express\urls.py", line 7, in <module>
     urlpatterns = patterns('',
 NameError: name 'patterns' is not defined
```

 I'm using Django=1.10.7, django-oscar=1.4 and django-oscar-paypal=0.9.7.

**Comment by artforlife** [comment]
@pisapapiros you need to take out the `patterns()` construct and substitute it with `[]`. That is, you `urlpatterns` variable should be a list.

**Comment by cglane** [comment]
Can we please integrate this change into the repository?

**Comment by solarissmoke** [comment]
I've addressed this and other issues in  #159.
